src/topology/algebra/infinite_sum.lean

```diff
@@ -456,6 +456,16 @@ end
 
 end tsum
 
+<<<<<<< HEAD
+=======
+/-!
+### Sums on subtypes
+
+If `s` is a finset of `α`, we show that the summability of `f` in the whole space and on the subtype
+`univ - s` are equivalent, and relate their sums. For a function defined on `ℕ`, we deduce the
+formula `(∑ i in range k, f i) + (∑' i, f (i + k)) = (∑' i, f i)`, in `sum_add_tsum_nat_add`.
+-/
+>>>>>>> upstream/master
```

Comment:
```suggestion
/-!
### Sums on subtypes

If `s` is a finset of `α`, we show that the summability of `f` in the whole space and on the subtype
`univ - s` are equivalent, and relate their sums. For a function defined on `ℕ`, we deduce the
formula `(∑ i in range k, f i) + (∑' i, f (i + k)) = (∑' i, f i)`, in `sum_add_tsum_nat_add`.
-/
```

Reply:
This was already fixed in 73e2a17, right?

Reply:
Hmm, than github is being stupid in showing me the diffs